test/classes/initializers/phase1/initFromThis.good

```diff
@@ -1,12 +1,12 @@
 initFromThis.chpl:6: In initializer:
-initFromThis.chpl:7: error: cannot take a reference to "this" before this.complete()
-initFromThis.chpl:9: error: cannot take a reference to "this" before this.complete()
-initFromThis.chpl:11: error: cannot initialize a variable from "this" before this.complete()
+initFromThis.chpl:7: error: cannot take a reference to "this" before "init this"
+initFromThis.chpl:9: error: cannot take a reference to "this" before "init this"
+initFromThis.chpl:11: error: cannot initialize a variable from "this" before "init this"
 initFromThis.chpl:15: In method 'init=':
-initFromThis.chpl:16: error: cannot take a reference to "this" before this.complete()
-initFromThis.chpl:18: error: cannot take a reference to "this" before this.complete()
-initFromThis.chpl:20: error: cannot initialize a variable from "this" before this.complete()
+initFromThis.chpl:16: error: cannot take a reference to "this" before "init this"
+initFromThis.chpl:18: error: cannot take a reference to "this" before "init this"
+initFromThis.chpl:20: error: cannot initialize a variable from "this" before "init this"
 initFromThis.chpl:28: In initializer:
-initFromThis.chpl:29: error: cannot take a reference to "this" before this.complete()
-initFromThis.chpl:31: error: cannot take a reference to "this" before this.complete()
-initFromThis.chpl:33: error: cannot initialize a variable from "this" before this.complete()
+initFromThis.chpl:29: error: cannot take a reference to "this" before "init this"
+initFromThis.chpl:31: error: cannot take a reference to "this" before "init this"
+initFromThis.chpl:33: error: cannot initialize a variable from "this" before "init this"
```

Comment:
You should not go down this rabbit hole today / in this PR, but someday we should unify error messages to use `'` rather than `"` for code segments we want to call out.